source/fe/fe_poly_tensor.cc

```diff
@@ -444,59 +455,59 @@ fill_fe_values (const Mapping<dim,spacedim>                      &mapping,
            ||
            ((mapping_type == mapping_piola) || (mapping_type == mapping_raviart_thomas)
             || (mapping_type == mapping_nedelec))))
-        switch (mapping_type)
-          {
-          case mapping_none:
-          {
-            for (unsigned int k=0; k<n_q_points; ++k)
-              for (unsigned int d=0; d<dim; ++d)
-                output_data.shape_values(first+d,k) = fe_data.shape_values[i][k][d];
-            break;
-          }
+        {
+          std::vector<Tensor<1,spacedim> > &transformed_shape_values = fe_data.transformed_shape_values;
```

Comment:
I would prefer if you simply used the original name of the variable below,rather than the alias you introduce here. Code with aliases for variables is difficult to read because it's not obvious if two different variable references may actually refer to the same variable.